rest_framework_simplejwt/backends.py

```diff
@@ -54,5 +54,7 @@ def decode(self, token, verify=True):
             return jwt.decode(token, self.verifying_key, algorithms=[self.algorithm], verify=verify,
                               audience=self.audience, issuer=self.issuer,
                               options={'verify_aud': self.audience is not None})
+        except InvalidAlgorithmError as ex:
+            raise TokenBackendError(_('Invalid algorithm specified')) from ex
```

Comment:
In the traceback, what is actually shown? Does JWT specifically say you need cryptography? Or is it simply an invalid algorithm? I can only find it here: https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jws.py

Reply:
There's two different messages on the original Error depending on why it was raised, "The specified alg value is not allowed" (for case 1 in my PR description) and "Algorithm not supported" for case 2